src/main/features/ccj/views/check-and-send.njk

```diff
@@ -12,20 +12,38 @@
 
     {{ tableStart("theirDetails", "Their details") }}
     {{ row('Full name:', details.defendant.name.name) }}
-    {{
-      rowWithMultipleValue("Address:", [
-        details.defendant.partyDetails.address.line1,
-        details.defendant.partyDetails.address.line2,
-        details.defendant.partyDetails.address.city,
-        details.defendant.partyDetails.address.postcode
-      ], addressUrl)
-    }}
+    {{ rowWithMultipleValue("Address:", [
+    details.defendant.partyDetails.address.line1,
+    details.defendant.partyDetails.address.line2,
+    details.defendant.partyDetails.address.city,
+    details.defendant.partyDetails.address.postcode
+    ], addressUrl) }}
     {{ row("Email:", details.defendant.email.address) }}
     {{ tableEnd() }}
 
     {{ tableStart("payment", "Payment") }}
     {{ row("Amount to be paid by defendant", amountToBePaid | numeral) }}
-    {{ row("When you want the defendant to pay", t(details.paymentOption.option.displayValue), paidAmountUrl) }}
+
+    {% if details.paymentOption.option.value == 'immediately' %}
+      {% set paymentOptionDisplayValue = [details.paymentOption.option.displayValue] %}
+    {% endif %}
+
+    {% if details.paymentOption.option.value == 'full' %}
+      {# Add date to pay by once the code is on master #}
+      {% set paymentOptionDisplayValue = [details.paymentOption.option.displayValue] %}
+
+    {% endif %}
+
+    {% if details.paymentOption.option.value == 'instalments' %}
+      {% set paymentOptionDisplayValue =
+      [ t(details.paymentOption.option.displayValue + ': %s '
```

Comment:
shouldn't we use 

```
t('%s: %s %s.', { ...})
```

Reply:
done

Reply:
not sure if its the nicest, because the display value's need translating, have pushed to show what it looks like